src/api/environments.ts

```diff
@@ -56,15 +55,12 @@ export const createEnvironment = async (
     project_id: string,
     params: CreateEnvironmentParams
 ): Promise<Environment> => {
-    const url = new URL(`/v1/projects/${project_id}/environments`, BASE_URL)
-    const response = await axios.post(url.href,
-        params,
-        {
-            headers: {
-                'Content-Type': 'application/json',
-                Authorization: token,
-            },
-        })
+    const response = await apiClient.post(`/v1/projects/${project_id}/environments`, params, {
+        headers: {
+            'Content-Type': 'application/json',
+            Authorization: token,
+        },
```

Comment:
doesn't necessarily need to be handled in this PR, but it'd be great to move this to the api client so that we don't need to add these headers to every request